addon-scheduler/src/main/java/org/vaadin/stefan/fullcalendar/Resource.java

```diff
@@ -118,11 +117,11 @@ public Resource(String id, String title, String color, Collection<Resource> chil
      * @param children children (optional)
      * @param businessHours businessHours (optional)
      */
-    public Resource(String id, String title, String color, Collection<Resource> children, BusinessHours businessHours) {
+    public Resource(String id, String title, String color, Collection<Resource> children, BusinessHours... businessHours) {
         this.id = id != null ? id : UUID.randomUUID().toString();
         this.title = title;
         this.color = color;
-        this.businessHours = businessHours;
+        this.businessHoursArray = businessHours;
```

Comment:
This one should check, if the array is empty and then set business hours array to null. 

Alternatively, if an explicit non null value is wanted for this field, the "toJson" conversion needs to check for an empty array, too. Otherwise an empty array is passed to the client all the time and it might change client side logic.

Reply:
Added getter